src/main/java/com/example/bytex/user/UserEntity.java

```diff
@@ -0,0 +1,52 @@
+package com.example.bytex.user;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table
```

Comment:
Usually it's better if you add the table name here. When naming tables, best practice is to use the singular form (e.g. `user` and not `users`). Because `user` is a reserved keyword in DB, you can use `[]` to escape:

```
@Table(name = "[user]")
```

Reply:
done